arch/arm/src/stm32f7/stm32_sdmmc.c

```diff
@@ -1223,9 +1237,25 @@ static uint8_t stm32_log2(uint16_t value)
 static void stm32_dataconfig(struct stm32_dev_s *priv, uint32_t timeout,
                              uint32_t dlen, uint32_t dctrl)
 {
-  uint32_t regval = 0;
+  uint32_t clkdiv;
```

Comment:
```suggestion
  uint32_t clkdiv   = 0;
```
nit: same as the previous one

Reply:
"